lib/Ed25519KeyPair.js

```diff
@@ -341,15 +434,23 @@ function ed25519VerifierFactory(key) {
 }
 
 /**
- * Wrap Base58 decoding operations in order to provide consistent error
- * messages.
- *
- * @param {function} decode - the decode function to use.
- * @param {string} keyMaterial - the Base58 encoded key material to decode.
- * @param {string} type - a description of the keyMaterial that will be included
- *   in an error message (e.g. 'public', 'private').
+ * @example
+ * const pubkeyBytes = _base58Decode({
+ *   decode: base58.decode,
+ *   keyMaterial: this.publicKeyBase58,
+ *   type: 'public'
+ *  });
+ * @description Wraps Base58 decoding operations in
```

Comment:
Why the switch here and elsewhere of using the `@description` tag vs just having the easier to read description at the top w/o a tag?

Reply:
@davidlehn experiment here is with the [js-doc linter plugin](https://github.com/digitalbazaar/eslint-config-digitalbazaar/pull/19) which demands an @description tag. So we can change the linter or not. One thing I did find is this the undeclared description for classes actually fills both @description and @classdesc tags which means we do lose the freedom to have snappy @classdesc at the top of the file and more in depth @descriptions in the actual class documentation.

Reply:
@mattcollier @dlongley @JoshDunnDev @dmitrizagidulin Can we get a call here? should we explicitly set the @description tag or should be use the inferred style?

Reply:
I think the existing inferred style is much easier to read. I don't understand the "snappy" use case you're talking about for the other method. Can you give an example?

Reply:
@dlongley @dlongley prefers inferred too so I am going to remove require @description from the js-doc lint rules and remove @description from all docs.